docs/Basics/Download-Project.md

```diff
@@ -0,0 +1,12 @@
+# Download a Project 
```

Comment:
Need to indicate somewhere that this is web only.

Also, file name should be downcase: `download-project.md`.